gui/src/api.js

```diff
@@ -3,17 +3,18 @@ import url from 'url'
 import { camelizeKeys } from 'humps'
 import 'isomorphic-unfetch'
 
-const DEFAULT_CHAINLINK_PORT = 6688
```

Comment:
We need to keep this so that we can use webpack dev server during development which serves the assets from `:3000`

Reply:
Discussed: will use `CHAINLINK_PORT`